deploy/operator.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: cluster-logging-operator
+  annotations:
+    clusterlogging.alpha/allinone: ""
```

Comment:
I think this needs to be namespaced similar to other annotations (i.e build) like: `io.openshift.clusterlogging.alpha` or somethign like that.